<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="address_init">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>975</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/upload_video/image/upload_video/address.png);
font: 36pt &quot;MS PGothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="placeholderText">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_init">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>140</y>
      <width>152</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/upload_video/image/upload_video/browse.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="upload">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>670</y>
      <width>381</width>
      <height>73</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/upload_video/image/upload_video/UPLOAD.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>150</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/upload_video/image/upload_video/initialize.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>150</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/upload_video/image/upload_video/exercise.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>259</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/upload_video/image/upload_video/exercise_type.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>259</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/upload_video/image/upload_video/type_rectangle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;KoPub돋움체 Bold&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Shoulder Press</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pull Up</string>
     </property>
    </item>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/overall/image/back.png);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="address_ex">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>975</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/upload_video/image/upload_video/address.png);
font: 36pt &quot;MS PGothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="placeholderText">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_ex">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>350</y>
      <width>152</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/upload_video/image/upload_video/browse.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>address_init</zorder>
   <zorder>browse_init</zorder>
   <zorder>upload</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>address_ex</zorder>
   <zorder>browse_ex</zorder>
   <zorder>comboBox</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
